### ISSUE TITLE
Postgresql 14 unsupported

### ISSUE THREAD

**Issue by kallisti5** [usefathom/fathom#341]
```
opening database: opening db: ping Postgres DB: pq: unknown authentication response: 10
```

Looks like the database provider for fathom needs updated to support newer versions of postgres.

**Comment by kallisti5** [comment]
Oof.. Looks like the postgresql mod is from 2018.
https://github.com/usefathom/fathom/blob/master/go.mod#L15

**Comment by kallisti5** [comment]
https://github.com/usefathom/fathom/pull/342 was opened with a solution

**Comment by LKaemmerling** [comment]
Fixed with #342
